cybergis/general/comet.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+
+from .base import SBatchScript
+from .job import SlurmJob
+from .connection import SSHConnection
+
+
+logger = logging.getLogger("cybergis")
+
+
+class CometSBatchScript(SBatchScript):
+    pass
+
+
+class CometJob(SlurmJob):
+
+    JOB_ID_PREFIX = "Comet_"
+    backend = "Comet"
+    connection_class = SSHConnection
+    sbatch_script_class = CometSBatchScript
+
+    def prepare(self):
```

Comment:
@CarnivalBug If this function is same as its baseclass (SlurmJob: prepare()), you can safely remove it here as it is inherited from the base.